woohoo.rb

```diff
@@ -0,0 +1,11 @@
+(1..10).reduce(&:+)
+(1..10).inject(&:+)
+
+'super long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long line'
+"super long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long linesuper long line"
+
+(1..50).reduce(&:+)
+(1..50).inject(&:+)
+
+(1..100).detect {|i| i % 5 == 0 and i % 7 == 0 }
+(1..100).find {|i| i % 5 == 0 and i % 7 == 0 }
```

Comment:
Use `&&` instead of `and`.<br>Prefer `find` over `find`.<br>Space between { and | missing.